public/app/percona/shared/components/Form/FormElement/FormElement.tsx

```diff
@@ -0,0 +1,83 @@
+import React from 'react';
```

Comment:
This is not being used in the Settings.

Reply:
Done, thanks for noticing it,  wasn't used anywhere, just missed this file deletion